Investment/src/main/java/com/example/investment/api/investmentapi/infrastructure/RequesterImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.example.investment.api.investmentapi.infrastructure;
+
+import com.example.investment.api.investmentapi.controller.dto.ApiRequest;
+import com.example.investment.api.investmentapi.controller.dto.ApiResponse;
+import com.example.investment.api.investmentapi.infrastructure.exception.ApiServerException;
+import com.example.investment.api.investmentapi.service.Requester;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.*;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+
+@RequiredArgsConstructor
+@Component
+@Slf4j
+public class RequesterImpl implements Requester {
+    private static final String URL = "https://apis.data.go.kr/1160100/service/GetStockSecuritiesInfoService/getStockPriceInfo";
+    private static final String X_API_KEY = "6vkAL8btLG4h51Af2JRQ5LUXImyRng8XzbfVmG1K7n2Vw0sY0D6BZmVUkVghiGzoR5pAHvnKdL5Xmrtw2fXDZw==";
```

Comment:
API 키 값이 그대로 노출되어도 될까요? 누군가 저걸 탈취해서 악용할 여지가 있을 것 같은데 jasypt나 환경변수로 숨기는게 좋을 것 같아요

```suggestion
public class RequesterImpl implements Requester {

    private static final String URL = "https://apis.data.go.kr/1160100/service/GetStockSecuritiesInfoService/getStockPriceInfo";
    private static final String X_API_KEY = "6vkAL8btLG4h51Af2JRQ5LUXImyRng8XzbfVmG1K7n2Vw0sY0D6BZmVUkVghiGzoR5pAHvnKdL5Xmrtw2fXDZw==";
```